tripleblindmarket/private-kit
# Scrypt performance time and staging build settings

#### Description:

This PR will address 2 things:
Adding signpost logs to measure the performance of geohash+scrypt hashing.
Work on the build machines for these failed builds https://github.com/Path-Check/covid-safe-paths/actions?query=workflow%3ABuild+branch%3Adevelop


#### How to test:

View console logs for "org.pathcheck.covid-safepaths" performance category mesurements
The staging machines should successfully build the project.
